### DISCUSSION TITLE
GetThreadContext failed when using lime rebuild extension-webm windows command

### DISCUSSION THREAD

**Discussion by PlasmaOJ15** [KadeArchive/Kade-Engine#1204] [Q&A]
basically what the title says. i get the message GetThreadContext failed when using the lime rebuild extension-webm windows command. anyone know how to fix?

**Comment by SirErika** [comment]
Turn off your anti-virus before compiling. Worked for me when I had this problem.

**Reply by PlasmaOJ15** [reply]
It didnt fix. Also, not an issue with compiling, issue with fixing the extension-webm library
